fuzz/fuzz_targets/deserialize_amount.rs

```diff
@@ -0,0 +1,74 @@
+extern crate bitcoin;
+use std::str::FromStr;
+fn do_test(data: &[u8]) {
+    let data_str = String::from_utf8_lossy(data);
+
+    // signed
+    let samt = match bitcoin::util::amount::SignedAmount::from_str(&format!("{} BTC", data_str)) {
```

Comment:
no need to add ` BTC` here; the fuzzer is capable of it (and it will also find the other denominations)